mLRS/Common/hal/esp/rx-hal-dev-900-esp32.h

```diff
@@ -0,0 +1,111 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+//*******************************************************
+// hal
+//********************************************************
+
+//-------------------------------------------------------
+// ESP8285, ELRS GENERIC 900 RX
+//-------------------------------------------------------
+
+#define DEVICE_HAS_SINGLE_LED
+//#define DEVICE_HAS_NO_DEBUG
+//#define DEVICE_HAS_SERIAL_OR_DEBUG
+
+
+//-- Timers, Timing, EEPROM, and such stuff
+
+#define SYSTICK_TIMESTEP          1000
+#define SYSTICK_DELAY_MS(x)       (uint16_t)(((uint32_t)(x)*(uint32_t)1000)/SYSTICK_TIMESTEP)
+
+#define EE_START_PAGE             0
+
+
+//-- UARTS
+// UARTB = serial port
+// UART = output port, SBus or whatever
+// UARTC = debug port
+
+#define UARTB_USE_SERIAL1
+#define UARTB_BAUD                RX_SERIAL_BAUDRATE
+#define UARTB_TXBUFSIZE           RX_SERIAL_TXBUFSIZE
+#define UARTB_RXBUFSIZE           RX_SERIAL_RXBUFSIZE
+
+#define UARTC_USE_SERIAL
+#define UARTC_BAUD                115200
+
+
+//-- SX1: SX12xx & SPI
+#define SPI_FREQUENCY             10000000L
+#define MISO                      19
+#define MOSI                      27
+#define SCK                       IO_P5
+#define SX_NSS                    18
```

Comment:
pl lets try to be consistent, pl use also SPI_CS_IO (like in the stm32 hals) here too

Reply:
I think should also be SPI_MISO_IO, etc pp

Reply:
aren't these defaults?

Reply:
> aren't these defaults?

Not defaults and can be remapped to many different pins, so this is needed unlike ESP8266.  Pins: 
![image](https://github.com/olliw42/mLRS/assets/41841496/62d6f1a1-6bfc-4719-98af-8940fb56b6d0)

Reply:
THX !!!

Reply:
Done